### PR TITLE
Bump gcloud-in-go to 1.8.3.

### PR DESCRIPTION
**Pull request by spxtr**
yak yak yak, please give LGTM then I'll push then we can merge.

### CODE REVIEW

**Comment by krzyzacy** [comment]
welp..

/lgtm

File: images/gcloud/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.1
+VERSION = 0.2
```
Comment (ixdy) [inline images/gcloud/Makefile]: we might consider using something like the go version so it's more obvious?

Comment (krzyzacy) [inline images/gcloud/Makefile]: maybe gcloud + go

Comment (spxtr) [inline images/gcloud/Makefile]: It's possible to do `goversion-gcloudversion` or something. I don't think it's super important though, just look at the git history...

Comment (ixdy) [inline images/gcloud/Makefile]: yeah, but we didn't realize until just now that 0.1 was a pretty old go + gcloud. if it was in the tag it'd be self-documenting and way more obvious.

**Pull request merged**